pages/linux/postfix.md

```diff
@@ -0,0 +1,28 @@
+# postfix
+
+> Postfix mail transfer agent (MTA) control program.
+> More information: <http://postfix.org>
+
+- Check the configuration:
+
+`sudo postfix check`
+
+- Check the status of the Postfix daemon:
+
+`sudo postfix status`
+
+- Start Postfix:
+
+`sudo postfix start`
+
+- Gracefully stop Postfix:
```

Comment:
How does one gracefully stop a program? Asking for a friend. Perhaps this should be changed, as @schneiderl doesn't like adjectives and quite possibly adverbs as well.

Reply:
Haha I’m actually ok with using Gracefully here. 
It only means that everything is shut down without errors not that it does anything better then any other command that does the same job.

Reply:
Gracefully is a term here that's used to describe the fact that it will first finish up all remaining jobs in the queue before shutting down.

If a program is not gracefully shut down, then it will stop and shut down before finishing up any remaining jobs. For instance, a HTTP server might shut down ungracefully after sending half a response and forcefully terminate all connections (or just leave them hanging O.o).

Reply:
`Stop Postfix after finishing all jobs` may be easier to understand in that case. What do you think?

Reply:
gracefully is more of a technical term than an adjective. It's meaning is what @sbrl explained above. We can go with "after finishing all jobs", but some pages already use this term. So if we are not using "gracefully" here, we should remove it from all other pages.

Reply:
I think we should keep `Gracefully` here. As @agnivade says, it is a technical term - so I'm very reluctant to change it.

Reply:
I understand that, I'm just asking whether it agrees with the general philosophy of the project. Are we trying to write hacker-style cheatsheets or documents accessible to the general public?

I had no clue of the meaning of "gracefully" in this context, but maybe the people who generally use the command do know what it means.